.github/PULL_REQUEST_TEMPLATE/new_post.md

```diff
@@ -0,0 +1,13 @@
+This PR adds a new post:
+
+# Content
+
+<!-- Short abstract/description of the post -->
+
+
+# Checklist
+
+ 1. [ ] No external resources (Check `README.md` how to add assets)
+ 2. [ ] References and citations are correct
+ 3. [ ] Commits `squash`ed, so that the post is contained within one commit
```

Comment:
I think this entry here needs a bit more consideration, would love to hear other opinions. IMHO it makes sense to have the commits that add new posts `squash`ed, as it prevents a completely cluttered git history.

Reply:
Totally agree. Could be done during merge aswell

Reply:
Let's do it on merge if it's not done by the PR creator.